kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/dto.scala

```diff
@@ -122,8 +120,6 @@ case class BatchRequest(
     resource: String,
     proxyUser: String,
     className: String,
-    jars: java.util.List[String],
-    files: java.util.List[String],
     name: String,
     conf: Map[String, String],
     args: java.util.List[String])
```

Comment:
Seq?

Reply:
`Seq` should be fine.

I just saw that some`dto` classes use `java.util.List`.